datalad/tests/test_utils_testrepos.py

```diff
@@ -33,12 +34,10 @@ def _test_BasicAnnexTestRepo(repodir):
     ok_file_under_git(trepo.path, 'test.dat')
     ok_file_under_git(trepo.path, 'INFO.txt')
     ok_file_under_git(trepo.path, 'test-annex.dat', annexed=True)
-    if not trepo.repo.is_crippled_fs():
-        ok_broken_symlink(pathjoin(trepo.path, 'test-annex.dat'))
+    ok_(trepo.repo.file_has_content('test-annex.dat') is False)
```

Comment:
I have got surprised that this line marked not covered... well -- for me it just fails altogether:

```shell
(git)hopa:~datalad/datalad[nf-direct-mode]git
$> git describe
0.4-80-gec22586
changes on filesystem:                                                                                          
 datalad/tests/test_utils_testrepos.py | 1 +
(venv-tests) 2 16182.....................................:Fri 28 Oct 2016 08:16:26 PM EDT:.
(git)hopa:~datalad/datalad[nf-direct-mode]git
$> nosetests datalad/tests/test_utils_testrepos.py
E.E
======================================================================
ERROR: datalad.tests.test_utils_testrepos.test_BasicAnnexTestRepo_random_location_generated
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/yoh/proj/datalad/datalad/datalad/tests/test_utils_testrepos.py", line 47, in test_BasicAnnexTestRepo_random_location_generated
    _test_BasicAnnexTestRepo(None)  # without explicit path -- must be generated
  File "/home/yoh/proj/datalad/datalad/datalad/tests/test_utils_testrepos.py", line 28, in _test_BasicAnnexTestRepo
    trepo = BasicAnnexTestRepo(repodir)
  File "/home/yoh/proj/datalad/datalad/datalad/tests/utils_testrepos.py", line 44, in __init__
    self.repo = self.REPO_CLASS(path)
  File "/home/yoh/proj/datalad/datalad/datalad/support/annexrepo.py", line 155, in __init__
    create=create, git_opts=git_opts)
  File "/home/yoh/proj/datalad/datalad/datalad/support/gitrepo.py", line 492, in __init__
    self.repo.git._persistent_git_options = self._GIT_COMMON_OPTIONS
AttributeError: 'Git' object has no attribute '_persistent_git_options'

```